src/Hamiltonians/correlation_functions.jl

```diff
@@ -236,3 +236,153 @@ function get_offdiagonal(
     gd = exp(-im*g.d*Δp*2π/M)*gamma
     return new_add, ComplexF64(gd/M)
 end
+
+"""
+    SuperfluidCorrelator(d::Int) <: AbstractHamiltonian{Float64}
+
+Two-body operator for superfluid correlation between sites separated by `d` with `0 ≤ d < m`
+
+```math
+    \\hat{C}_{\\text{SF}}(d) = \\frac{1}{M} \\sum_{i}^{M} a_{i}^{\\dagger} a_{i + d}
+```
+Assumes a one-dimensional lattice with periodic boundary conditions.
+
+SuperfluidCorrelator must be wrapped in 'AllOverlaps'
+
+# Arguments
+- `d::Integer`: distance between sites.
+
+# Implementation
+Follow below for an example on how 'SuperfluidCorrelator' is used. Implementation is similar to that of 'G2RealCorrelator'
+- [G2 Correlator Example](https://joachimbrand.github.io/Rimu.jl/previews/PR227/generated/G2-example.html)
+
+
+# See also 
+
+* [`HubbardReal1D`](@ref)
+* [`G2RealCorrelator`](@ref)
+* [`AbstractHamiltonian`](@ref)
+* [`AllOverlaps`](@ref)
+
```

Comment:
```suggestion
See also [`HubbardReal1D`](@ref), [`G2RealCorrelator`](@ref), [`AbstractHamiltonian`](@ref),
and [`AllOverlaps`](@ref).
```
More concise and following style convention.